### ISSUE TITLE
Magic sauce is missing. Why is the API not equivalent to  backend-api/conversation?

### ISSUE THREAD

**Issue by lstappen** [openai/openai-python#267]
### Describe the feature or improvement you're requesting

Thank you for your great work in providing an API. 
What seems strangely confusing is that the main use of a conversation_id is not part of the API. Not providing the main use of a chat context as a reference point leads to pointless data stored in memory, repeated requests within a chat, limitations regarding the no of tokens and unnecessary costs.

Since it is definitely technically feasible, as there is an API for the chat window, I wonder what the intention behind this is? Is it simply temporary? Before any workarounds are developed, and if so, when, is a full deployment planned? It also seems that other parties e.g. the companies of the demos have that ability?

Appreciate your guys work and thank you very much for your help!

**Comment by logankilpatrick** [comment]
> What seems strangely confusing is that the main use of a conversation_id is not part of the API

What do you mean here? The API does not manage the conversation history, this is something that needs to be built on the developer side and my guess is that people will come up with cool open source solutions, but by default, the API does not do this. I am going to close this for now, but stay tuned for any updates to the API.
